scripts/testing_scripts/update_lib.sh

```diff
@@ -1,6 +1,33 @@
 #!/bin/bash
 
 if [ "$1" = bowman ]; then
+  ICPCVER="$(icpc --version | grep 17 | cut -d ' ' -f 3)"
+  if [[ "${ICPCVER}" = 17.* ]]; then
```

Comment:
I think this needs to be for 17 and 18.  18 looks like the default is 4.9.3 as well

Reply:
Thanks for pointing that out, just pushed an update

Reply:
Made similar update for blake, 18.0 loads gcc/4.9.3 but 18.1 does not.